engine/components-api/src/main/java/com/cloud/configuration/ConfigurationManager.java

```diff
@@ -54,10 +55,30 @@
  */
 public interface ConfigurationManager {
 
-    public static final String MESSAGE_CREATE_POD_IP_RANGE_EVENT = "Message.CreatePodIpRange.Event";
-    public static final String MESSAGE_DELETE_POD_IP_RANGE_EVENT = "Message.DeletePodIpRange.Event";
-    public static final String MESSAGE_CREATE_VLAN_IP_RANGE_EVENT = "Message.CreateVlanIpRange.Event";
-    public static final String MESSAGE_DELETE_VLAN_IP_RANGE_EVENT = "Message.DeleteVlanIpRange.Event";
+    String MESSAGE_CREATE_POD_IP_RANGE_EVENT = "Message.CreatePodIpRange.Event";
+    String MESSAGE_DELETE_POD_IP_RANGE_EVENT = "Message.DeletePodIpRange.Event";
+    String MESSAGE_CREATE_VLAN_IP_RANGE_EVENT = "Message.CreateVlanIpRange.Event";
+    String MESSAGE_DELETE_VLAN_IP_RANGE_EVENT = "Message.DeleteVlanIpRange.Event";
+
+    ConfigKey<Boolean> SystemVMUseLocalStorage = new ConfigKey<>(
+            Boolean.class,
+            "system.vm.use.local.storage",
+            "Advanced",
+            "false",
+            "Indicates whether to use local storage pools or shared storage pools for system VMs.",
+            false,
+            ConfigKey.Scope.Zone,
+            null);
+
+    ConfigKey<String> ControlCidr = new ConfigKey<>(
```

Comment:
we need to add this config to getConfigKeys() method @DaanHoogland 
https://github.com/shapeblue/cloudstack/blob/1275db4081aa8b09148909d6e6b65bc377d4992f/server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java#L7630